src/foam/box/SharedWorkerRegistry.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
This file can be ignored for now. It will be updated once `DedicatedWorkerRegistry` is fixed.